dependencies.toml

```
@@ -159,6 +159,8 @@ tomcat8 = "8.5.98"
 tomcat9 = "9.0.85"
 tomcat10 = "10.1.18"
 xml-apis = "1.4.01"
+zipkin = "3.0.6"
+zipkin-reporter = "3.2.1"
```

since bumping, might as well!

```suggestion
zipkin-reporter = "3.3.0"
```

---

At let me remove these dependencies since they are not used. 😉